packages/database/src/migrations/20210317035935-AddConfigForCategoryHeadingLaosEoc-modifies-data.js

```diff
@@ -0,0 +1,151 @@
+'use strict';
+
+import { updateValues } from '../utilities/migration';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+async function getDashboardReportById(db, id) {
+  const { rows: dashboardReports } = await db.runSql(`
+      SELECT * FROM "dashboardReport"
+      WHERE id = '${id}';
+  `);
+  return dashboardReports[0] || null;
+}
+
+const categoryPresentationOptions = {
+  type: 'objectCondition',
+  conditions: [
+    {
+      key: 'red',
+      color: '#b71c1c',
+      condition: {
+        '=': null,
+      },
+      legendLabel: 'Stock out',
+    },
+    {
+      key: 'green',
+      color: '#33691e',
+      label: '',
+      condition: {
+        '>': 0,
+      },
+      legendLabel: 'In stock',
+    },
+    {
+      key: 'orange',
+      color: 'orange',
+      label: '',
+      condition: {
+        some: { '>': 0 },
+      },
+      legendLabel: 'At least 1 item out of stock',
```

Comment:
A much more specific comment here - the condition you've got isn't quite right. It is actually testing 'At least 1 item is _in_ stock'. 

I think what you want is
```suggestion
        some: { '=': null },
      },
      legendLabel: 'At least 1 item out of stock',
```

Reply:
Oh I see - it's only going to fire if both of the others don't pass, so it would work properly either way. You can ignore me here, but it might become a moot point if the condition checking moves to the back end anyway!